job-descriptions/software-engineer-security.md

```diff
@@ -0,0 +1,51 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Security
+
+We are looking for a hands-on software engineer to own the security our product. This includes the security sourcegraph.com as well as making it easy for our customers to secure their own private Sourcegraph instances that they deploy on-premise. You will be responsible for proactively understanding, documenting, and improving the security of our product, as well as promptly responding to and addressing security issues reported to us by third parties. As our security needs grow over time, there is a potential for you to grow into a security team leadership role.
```

Comment:
Are software engineers not hands-on? This feels odd to me. I would drop it and clarify what you're looking for in a later sentence, although I think the final sentence clarifies this already.